web/pandas/about/team.md

```diff
@@ -36,6 +36,14 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+## Diversity and Inclusion
+
+> _pandas_ expressly welcomes and encourages anyone who faces under-representation, discrimination in the technology industry
+> or anyone willing to increase the diversity of our team, we encourage you to contribute.
```

Comment:
It might be a matter of personal opinion, but I think the part about encouriging contributions is too far back in the sentence. I would change this phrase for something like this: `pandas expressly welcomes and encourages contributions from members of underrepresented groups in technology willing to increase the diversity of our team.`

Reply:
Makes total sense thank you @martinagvilas for your review will fix that

Reply:
Waiting for the rest of the ladies then I make a single commit